src/hiveboard_bridge/include/hiveboard_bridge/HiveBoardBridge.h

```diff
@@ -0,0 +1,56 @@
+#ifndef HIVEBOARD_BRIDGE_HIVEBOARDBRIDGE_H
+#define HIVEBOARD_BRIDGE_HIVEBOARDBRIDGE_H
+#include "hiveboard_bridge/HiveBoardBridgeImpl.h"
+#include "hiveboard_bridge/MessageHandler.h"
+#include "hiveboard_bridge/TCPServer.h"
+#include <hivemind-host/HiveMindHostDeserializer.h>
+#include <hivemind-host/HiveMindHostSerializer.h>
+#include <memory>
+
+/**
+ * The HiveBoardBridge class is the main entrypoint to the bridge's library. It includes everything
+ * necessary to define a custom swarm API and provides some hooks to control the robot when some
+ * usual events occur, such as connection or disconnection of a HiveBoard.
+ */
+class HiveBoardBridge {
```

Comment:
Maybe rename every reference from HiveBoard ->HiveMind since the hiveboard is only the hardware, search and replace in the project should do the job

Reply:
Actually, let me do just that in a separate PR since i'll have to rename basically everything